lib/ansible/modules/system/pids.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/python
+# Copyright 2019, Saranya Sridharan
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+module: pids
+version_added: 2.8
+description: "Retrieves a list of process IDs (PIDs) of all processes of the given name. Returns an empty list if no process of the given name exists."
```

Comment:
It would be nice to add a note - From where this module will gather information about PIDs viz., Ansible Controller or Managed nodes or remote machines.